functional-tests/src/test/groovy/specs/InformationPageSpecs.groovy

```diff
@@ -1,50 +0,0 @@
-/*
-    Licensed under the Apache License, Version 2.0 (the "License");
-    you may not use this file except in compliance with the License.
-    You may obtain a copy of the License at
-
-        http://www.apache.org/licenses/LICENSE-2.0
-
-    Unless required by applicable law or agreed to in writing, software
-    distributed under the License is distributed on an "AS IS" BASIS,
-    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    See the License for the specific language governing permissions and
-    limitations under the License.
-*/
-
-package specs
-
-import pages.InformationPage
-import pages.ExternalLinkPage
-
-import spock.lang.Timeout
-import spock.lang.Title
-import spock.lang.Narrative
-import spock.lang.Unroll
-
-@Timeout(60)
-@Title('Groundwater wells Flow Tests')
-@Narrative('''
-As a developer, I want to ensure all page links work, have the correct text, and direct to the correct page.
-''')
-class InformationPageSpecs extends BaseSpec {
-
-  @Unroll
-  void 'Navigate Page from: InformationPage, click page Link: #TextSelector, Assert Page: #AssertPage'() {
```

Comment:
These were moved to FlowSpecs.groovy to consolidate flow spec type tests in one place.